whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -1391,8 +1391,111 @@
 
     "relation_from_w": {
       "computeLinks": {"use": "$w", "mapping": {"g": "broader", "h": "narrower", "*": "related"}}
-    }
+    },
+
+    "referencedObjectDetails": {
+      "pendingResources": {
+        "_:contribution": {
+          "about": "_:work",
+          "addLink": "contribution",
+          "resourceType": "PrimaryContribution",
+          "embedded": true
+        },
+        "_:agent": {
+          "about": "_:contribution",
+          "link": "agent",
+          "resourceType": "Agent"
+        },
+        "_:workTitle": {
+          "about": "_:work",
+          "addLink": "hasTitle",
+          "resourceType": "Title"
+        },
+        "_:title": {
+          "about": "_:instance",
+          "addLink": "hasTitle", "resourceType": "Title",
+          "embedded": true
+        },
+        "_:describedBy": {
+          "NOTE": "If not possible to link, LinkFinder will change it to a Local identifiedBy property",
+          "about": "_:instance",
+          "addLink": "describedBy", "infer": true,
+          "resourceType": "Record"
+        }
+      },
+      "i1": {
+        "NOTE": "By moving 580 to 500 this indicator will not be usefull anymore",
+        "marcDefault": "0"
```

Comment:
Should we add ignored = true to indicators with default values to prevent any other values from showing up in unhandled?

Reply:
Good idea. I fix :)